<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplaySettings</class>
 <widget class="KittySDK::SettingPage" name="DisplaySettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Display</string>
  </property>
  <property name="name" stdset="0">
   <string notr="true">Display</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="displayGroupBox">
     <property name="title">
      <string>Main window</string>
     </property>
     <layout class="QFormLayout" name="_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="alwaysOnTopCheckBox">
        <property name="text">
         <string>Always on top</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="transparencyCheckBox">
        <property name="text">
         <string>Transparency:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="autoHideCheckBox">
        <property name="text">
         <string>Autohide after:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QSlider" name="autoHideSlider">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>120</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="transparencyMinLabel">
        <property name="text">
         <string notr="true">10%</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QSlider" name="transparencySlider">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="transparencyMaxLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string notr="true">100%</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="autoHideMinLabel">
        <property name="text">
         <string>0 sec</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="autoHideMaxLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>120 sec</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="dockingGroupBox">
     <property name="title">
      <string>Docking</string>
     </property>
     <layout class="QFormLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="2" column="0">
       <widget class="QLabel" name="dockDistanceLabel">
        <property name="text">
         <string>When closer than:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="dockDistanceSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> px</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="dockToEdgesCheckBox">
        <property name="text">
         <string>Dock to desktop edges</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="captionsGroupBox">
     <property name="title">
      <string>Captions</string>
     </property>
     <layout class="QGridLayout" name="_3">
      <item row="0" column="0">
       <widget class="QLabel" name="mainWindowCaptionLabel">
        <property name="text">
         <string>Main window:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="mainWindowCaptionEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="mainWindowCaptionHelpButton">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="chatWindowCaptionLabel">
        <property name="text">
         <string>Chat window:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="chatWindowCaptionEdit"/>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="chatWindowCaptionHelpButton">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="chatTabCaptionLabel">
        <property name="text">
         <string>Chat tab:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLineEdit" name="chatTabCaptionEdit"/>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="chatTabCaptionHelpButton">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KittySDK::SettingPage</class>
   <extends>QWidget</extends>
   <header>SDK/SettingPage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>autoHideCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>autoHideSlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>transparencyCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>transparencySlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dockToEdgesCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>dockDistanceSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
